dapps/src/endpoint.rs

```diff
@@ -18,6 +18,7 @@
 
 use hyper::{server, net};
 use std::collections::BTreeMap;
+pub use hyper::Control;
```

Comment:
I wanted to keep `Endpoints` decoupled from `hyper`. Maybe it would be better to have custom `Control` wrapper instead (perhaps with better name).